llvm/lib/CheerpUtils/ConstantExprLowering.cpp

```diff
@@ -35,39 +35,69 @@ bool ConstantExprLowering::runOnFunction(Function& F, bool& hasI64, const Target
 
 	std::set<ConstantExpr*> visitedCE;
 	std::deque<ConstantExpr*> toBeInstructionized;
+	std::vector<ConstantExpr*> CEvector;
```

Comment:
The diff is kind of fucked up, so maybe I am not seeing something. But why are we adding this vector?
It seems that we collect stuff here and then iterate on it to collect other stuff in the workList. Why can't we collect directly in the workList like before?